engine/image-loader/packages.lisp

```diff
@@ -2,8 +2,14 @@
 
 (defpackage :lbge.image-loader
   (:use :cl)
-  (:export :load-image :make-image))
+  (:export :load-image)
+  (:import-from :alexandria
+                :switch))
+
+(defpackage :lbge.image-loader.image
+  (:use :cl)
+  (:export :make-image :width :height :channels :data))
 
 (defpackage :lbge.image-loader.tga
-  (:use :cl :lbge.image-loader)
+  (:use :cl :lbge.image-loader.image)
```

Comment:
It seems to me that it would be nice (i.e. more logical) to rename main package to image, and make loader it's sub instead.
Of course it may be done as a separate issue.